restapi/src/main/java/io/stargate/web/resources/ResourceUtils.java

```diff
@@ -0,0 +1,21 @@
+package io.stargate.web.resources;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ResourceUtils {
+
+  public static Map<String, Object> readJson(String payload, ObjectMapper mapper) {
```

Comment:
Could now also move `ObjectMapper` as a static member here? That would allow another thing, see next comment....